.gitignore

```diff
@@ -1,9 +1,10 @@
 .gradle
 build
 .idea/
+*.iml/
 .project
 .settings/
 .classpath
 nbactions.xml
 nb-configuration.xml
-nbproject/
\ No newline at end of file
+nbproject/
```

Comment:
Please don't push your .gitignore file

Reply:
I saw there were other Intellij-related files in there, so I assumed this one was just missing. I'll restore the .gitignore from master.